sympy/physics/control/control_plots.py

```diff
@@ -736,18 +737,25 @@ def bode_magnitude_numerical_data(system, initial_exp=-5, final_exp=5, **kwargs)
 
 
 def bode_magnitude_plot(system, initial_exp=-5, final_exp=5,
-    color='b', show_axes=False, grid=True, show=True, **kwargs):
+    color='b', show_axes=False, grid=True, show=True, Freq_unit='rad/sec', **kwargs):
     r"""
     Returns the Bode magnitude plot of a continuous-time system.
 
     See ``bode_plot`` for all the parameters.
     """
+    Freq_units = ('rad/sec', 'Hz')
+    if Freq_unit not in Freq_units:
+        raise ValueError('Only rad/sec and Hz are accepted frequency units.')
+
     x, y = bode_magnitude_numerical_data(system, initial_exp=initial_exp,
         final_exp=final_exp)
+    if Freq_unit == 'Hz':
+        a = S.Pi.n()
+        x = [i/(2*(a)) for i in x]
```

Comment:
```suggestion
        x = [i/(2*a) for i in x]
```
(Although better to move it.)